packages/fetchai/connections/soef/connection.yaml

```diff
@@ -6,7 +6,7 @@ license: Apache-2.0
 aea_version: '>=0.3.0, <0.4.0'
 fingerprint:
   __init__.py: Qmd5VBGFJHXFe1H45XoUh5mMSYBwvLSViJuGFeMgbPdQts
-  connection.py: QmRTUx1SSuKGrD77MKPYV8oc4J2G6vW92cJMcEXaijDXUc
+  connection.py: QmePFpSeVgKnyi5kprB6dQz8fg9kgJpxeJ59PaxJYuKPd4
```

Comment:
When we update code, for the first time following a release we need to bump the version number of the package.